hosting/photo-storage/README.md

```diff
@@ -0,0 +1,59 @@
+# Photo storage example
+
+The example shows how to store photos on the IC in an asset canister with the `@dfinity/assets` package. The photo storage app is deployed as frontend 
+in an assets canister which is also used for photo upload.
+
+## Installation
+This example project can be cloned, installed and deployed locally, for learning and testing purposes. The instructions are based on running the example on either macOS or Linux, but when using WSL2 on Windows, the instructions will be the same.
+
+### Prerequisites
+The example project requires the following installed:
+
+- git
+- dfx
+- npm
+
+git can be installed from various package managers. DFX can be installed following the instructions [here](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove).
+
+### Download the code
+Clone the example dapp project:
+
+```bash
+$ git clone https://github.com/dfinity/examples
+$ cd examples/hosting/photo-storage
+```
+
+## React build
+The React frontend is build by running this command:
+```bash
+npm install --legacy-peer-deps
+npm run build
+```
+`--legacy-peer-deps` flags is used here since `react-masonry-component` is not updated yet to have React 18 as peer dependency.
+
+## Deployment
+The local network is started by running this command:
+
+```bash
+$ dfx start --background
+```
+
+When the local network is up and running, run this command to deploy the canisters:
+
+```bash
+$ dfx deploy
+```
+
+## Authorization
+To authorize the identity from this example project on a local network to upload files, it must be authorized first:
+```bash
+dfx canister call photo-storage authorize '(principal "535yc-uxytb-gfk7h-tny7p-vjkoe-i4krp-3qmcl-uqfgr-cpgej-yqtjq-rqe")'
+```
```

Comment:
If you could add some sort of feedback if this gets forgotten that would be great! Even a simple `alert("please follow instructions in ##Authorization")` (or whatever the syntax in JS is for that) would be very helpful.